src/cirrusTheme.ts

```diff
@@ -56,15 +57,9 @@ let cirrusTheme = {
       hover: cirrusColors.cirrusGrey,
     },
   },
-  overrides: {
```

Comment:
https://github.com/mui-org/material-ui/pull/17584 broke this so need to find another solution